ps_viewedproduct.php

```diff
@@ -290,4 +295,16 @@ protected function getViewedProducts()
 
         return false;
     }
+
+    private function getExistingProductsIds()
```

Comment:
Doc block? :trollface:

Reply:
this will make the pull request unreadable!

Reply:
You can do it bro!

Reply:
Maybe the PHPDoc can be added in an 2nd PR ? So we have PR 1 that modifies code behavior, no refactoring ; and PR 2 that only adds harmless doc